addons/account/static/src/js/account_followup_report_widgets.js

```diff
@@ -0,0 +1,160 @@
+        var model = new openerp.Model('account.move.line');
+        model.call('write', [[parseInt(target_id)], {'blocked': checkbox}])
+    },
+    onKeyPress: function(e) {
+        var report_name = $("div.page").attr("class").split(/\s+/)[2];
+        if ((e.which === 13 || e.which === 10) && (e.ctrlKey || e.metaKey) && report_name == 'followup_report') {
+            $('a.btn-primary.followup-email').trigger('click');
+            var letter_context_list = [];
+            $('a.btn-primary.followup-letter').each(function() {
+                letter_context_list.push($(this).attr('context'))
+            });
+            window.open('?pdf&letter_context_list=' + letter_context_list, '_blank');
+            window.open('?partner_done=all', '_self');
+        }
+    },
+    skipPartner: function(e) {
+        var partner_id = $(e.target).attr("partner");
+        var model = new openerp.Model('res.partner');
+        if ($(e.target).attr('class') == 'btn btn-primary followup-skip') {
+            model.call('update_next_action', [[parseInt(partner_id)]]).then(function (result) {
+                window.open('?partner_done=' + partner_id, '_self');
+            });
+        }
+        else {
+            window.open('?partner_skipped=' + partner_id, '_self');
+        }
+    },
+    printFollowupLetter: function(e) {
+        e.stopPropagation();
+        e.preventDefault();
+        var url = $(e.target).attr("target");
+        window.open(url, '_blank');
+        if ($(e.target).attr("class").split(/\s+/)[1] == 'btn-primary') {
+            var $skipButton = $(e.target).siblings('a.followup-skip');
+            $skipButton.attr('class', 'btn btn-primary followup-skip');
+            $skipButton.text('Done');
+            $(e.target).attr('class', 'btn btn-default followup-letter');
+        }
+    },
+    sendFollowupEmail: function(e) {
+        e.stopPropagation();
+        e.preventDefault();
+        var context_id = $(e.target).parents("div.page").attr("class").split(/\s+/)[3];
+        var contextModel = new openerp.Model('account.report.context.followup');
+        contextModel.call('send_email', [[parseInt(context_id)]]).then (function (result) {
+            if (result == true) {
+                window.$("div.page:first").prepend(openerp.qweb.render("emailSent"));
+                if ($(e.target).attr("class").split(/\s+/)[1] == 'btn-primary') {
+                    var $skipButton = $(e.target).siblings('a.followup-skip');
+                    $skipButton.attr('class', 'btn btn-primary followup-skip');
+                    $skipButton.text('Done');
+                    $(e.target).attr('class', 'btn btn-default followup-email')
+                }
+            }
+            else {
+                window.$("div.page:first").prepend(openerp.qweb.render("emailNotSent"));
+            }
+        });
+    },
+    displayExpNoteModal: function(e) {
+        e.stopPropagation();
```

Comment:
I think that stopPropagation and preventdefault are not needed, if you have problem with multiple handler calling this event, it is better to find why you have mutliple handler in the first place